heaps/heap_sort.py

```diff
@@ -1,8 +1,16 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n log n)
+        Space Complexity: O(n)
     """
-    pass
\ No newline at end of file
+    heap = MinHeap()
+
+    for item in list:
+        heap = heap.add(item)
```

Comment:
Your failing tests for `heap_sort` return `AttributeError: 'NoneType' object has no attribute 'add'`. Look at the return value for `add` and see if you can figure out why that might be 🤔

What are you setting the heap equal to here? Do you need to be?